src/pages/Lending/Pool/components/Borrow/BorrowConfirm.tsx

```diff
@@ -292,7 +292,7 @@ export const BorrowConfirm = ({
     const maybeTxId = await (() => {
       // TODO(gomes): isTokenDeposit. This doesn't exist yet but may in the future.
       const sendInput: SendInput = {
-        cryptoAmount: depositAmount ?? '0',
+        amountCryptoPrecision: depositAmount ?? '0',
```

Comment:
`depositAmount` too?

Reply:
This change goes deep through Lending - I'm going to omit it from this PR.